Makefile

```diff
@@ -102,28 +102,38 @@ test-e2e-sno-disruptive: test-unit
 # -------------------------------------------------------------------
 .PHONY: tests-ext-build
 tests-ext-build:
-	$(MAKE) -C test/extended/tests-extension build
+	@mkdir -p $(TESTS_EXT_OUTPUT_DIR)
+	CGO_ENABLED=0 go build -ldflags "$(TESTS_EXT_LDFLAGS)" -o $(TESTS_EXT_OUTPUT_DIR)/$(TESTS_EXT_BINARY) $(TESTS_EXT_DIR)
 
 # -------------------------------------------------------------------
 # Run "update" and strip env-specific metadata
 # -------------------------------------------------------------------
 .PHONY: tests-ext-update
-tests-ext-update:
-	$(MAKE) -C test/extended/tests-extension build-update
+tests-ext-update: tests-ext-build
+	$(TESTS_EXT_OUTPUT_DIR)/$(TESTS_EXT_BINARY) update
 
 # -------------------------------------------------------------------
 # Clean test extension binaries
 # -------------------------------------------------------------------
 .PHONY: tests-ext-clean
 tests-ext-clean:
-	$(MAKE) -C $(TESTS_EXT_DIR) clean
+	rm -f $(TESTS_EXT_OUTPUT_DIR)/$(TESTS_EXT_BINARY)
```

Comment:
instead of creating a new target for removing the test binary could we reuse already existing `clean` target ?